doc/reference/tooling/tt_cli/cluster.rst

```diff
@@ -271,6 +272,74 @@ to ``ro`` and reloads the configuration.
     replica set members, so there can be any number of read-write instances in one replica set.
 
 
+.. _tt-cluster-failover:
+
+failover
+--------
+
+.. code-block:: console
+
+    $ tt cluster failover SUBCOMMAND [OPTION ...]
+
+``tt cluster failover`` manages a :ref:`supervised failover <repl_supervised_failover>` in Tarantool clusters.
+
+-   :ref:`switch <tt-cluster-failover-switch>`
+-   :ref:`switch-status <tt-cluster-failover-switch-status>`
+
+.. important::
+
+    ``tt cluster failover`` works only with centralized cluster configurations stored in etcd.
+
+
+.. _tt-cluster-failover-switch:
+
+switch
+~~~~~~
+
+.. code-block:: console
+
+    $ tt cluster failover switch URI INSTANCE_NAME [OPTION ...]
+
+``tt cluster failover switch`` appoints the specified instance to be a master.
+This command accepts the following arguments and options:
+
+-   ``URI``: A :ref:`URI <tt-cluster-uri>` of the cluster configuration storage.
```

Comment:
I found a possible issue that applies to the whole page.
In the `tt cluster` reference, the `URI` arguments refers to the _config storage URI_, while in the rest of the `tt` reference it's the _instance URI_. This can confuse readers. Perhaps we should change the argument name on this page to `CONFIG_URI` or smth like that.

Reply:
Renamed the `URI` argument to `CONFIG_URI`